scss/detail.scss

```diff
@@ -9,12 +9,23 @@
 }
 
 .book-cover-page {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-shadow;
   display: flex;
-  margin: 50px 0 0 50px;
   height: 416px;
+  margin: 50px 0 0 50px;
   width: 900px;
-  background-color: $white;
-  box-shadow: 4px 4px 10px 0 $black-shadow;
+}
+
+.book-cover-page::after {
```

Comment:
nice but you can made pseudo element inside of book-cover-page something like:
`
.book-cover-page {
  background-color: $white;
  box-shadow: 4px 4px 10px 0 $black-shadow;
  display: flex;
  height: 416px;
  margin: 50px 0 0 50px;
  width: 900px;

  &::after {
    background: url('images/badge.png') no-repeat;
    content: '';
    height: 120px;
    left: 250px;
    position: absolute;
    top: 50px;
    transform: rotate(1deg);
    width: 100px;
  }
}
`